src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -956,6 +959,11 @@ export interface AddVestingEscrowWalletParams extends AddModuleParams {
   data: VestingEscrowWalletData;
 }
 
+interface AddAdvancedPLCRVotingCheckpointParams extends AddModuleParams {
```

Comment:
Not necessary if the module requires no data, already covered by `AddNoDataModuleParams`